pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SerDe.scala

```diff
@@ -282,3 +282,9 @@ object SerDe {
     value.foreach(v => writeBytes(out, v))
   }
 }
+
+object SerializationFormats {
```

Comment:
BTW regardless of what we decide on the above proposal - let's be consistent in naming and change `SerializationFormats` to the name on R side (`SerializedModes` or `SerializationModes`).